From 5acf1735f7f497662c6ebe43220489dc81785ec6 Mon Sep 17 00:00:00 2001
From: Michael Palimaka <kensington@gentoo.org>
Date: Wed, 9 Jul 2014 23:54:11 +1000
Subject: [PATCH] Don't search default paths when finding lconvert.

Otherwise, if lconvert exists in normal system paths (eg. /usr/bin) that one
will be used instead of the one alongside Qt5::lrelease. This could cause Qt4
lconvert to be incorrectly used on some systems.

REVIEW: 119198
---
 modules/ECMCreateQmFromPoFiles.cmake | 1 +
 modules/ECMPoQmTools.cmake           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/modules/ECMCreateQmFromPoFiles.cmake b/modules/ECMCreateQmFromPoFiles.cmake
index 4a31a93..b66e598 100644
--- a/modules/ECMCreateQmFromPoFiles.cmake
+++ b/modules/ECMCreateQmFromPoFiles.cmake
@@ -113,6 +113,7 @@ function(_ECM_QM_CREATE_TARGET install_destination catalog_name)
     find_program(lconvert_executable
         NAMES lconvert-qt5 lconvert
         PATHS ${lrelease_path}
+        NO_DEFAULT_PATH
         )
 
     if (catalog_name)
diff --git a/modules/ECMPoQmTools.cmake b/modules/ECMPoQmTools.cmake
index 3ce6958..74dc656 100644
--- a/modules/ECMPoQmTools.cmake
+++ b/modules/ECMPoQmTools.cmake
@@ -129,6 +129,7 @@ function(ecm_process_po_files_as_qm lang)
     find_program(lconvert_executable
         NAMES lconvert-qt5 lconvert
         PATHS ${lrelease_path}
+        NO_DEFAULT_PATH
         )
 
     # Create commands to turn po files into qm files
-- 
1.8.5.5

